patches/tools-git-move_source_file.py.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/tools/git/move_source_file.py b/tools/git/move_source_file.py
+index ab57ae721179f77bb92b808e65dfe1ced8cf605f..4536e3c8d8def51553be568f4332802a936f30e7 100755
+--- a/tools/git/move_source_file.py
++++ b/tools/git/move_source_file.py
+@@ -30,7 +30,9 @@ if __name__ == '__main__':
+   # Need to add the directory containing sort_sources.py to the Python
+   # classpath.
+   sys.path.append(os.path.abspath(os.path.join(sys.path[0], '..')))
++  sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir, 'brave', 'script'))
```

Comment:
we can add this to the python path by calling it through an npm command

Reply:
done.